src/olympia/reviews/serializers.py

```diff
@@ -0,0 +1,33 @@
+from rest_framework import serializers
+
+from olympia.reviews.models import Review
+from olympia.users.serializers import BaseUserSerializer
+
+
+class BaseReviewSerializer(serializers.ModelSerializer):
+    # title and body are TranslatedFields, but there is never more than one
+    # translation for each review - it's essentially useless. Because of that
+    # we use simple charfields in the API, hiding the fact that it's a
```

Comment:
we can, it's going to be super annoying to migrate, but it's probably a good idea. I'll file an issue.